experiments/ClimaCore/sea-ice/slab_ice.jl

```diff
@@ -0,0 +1,341 @@
+# # Heat Equation + Slab Tutorial
+
+#  - load external packages:
+import LinearAlgebra, UnPack
+import ClimaCore: Fields, Domains, Topologies, Meshes, DataLayouts, Operators, Geometry, Spaces
+
+using Base: show_supertypes
+using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
+
+using Logging: global_logger
+using TerminalLoggers: TerminalLogger
+
+using RecursiveArrayTools
+
+using OrdinaryDiffEq
+
+using Statistics
+
+#src ## Setup Logging Information
+global_logger(TerminalLogger())
+const CI = !isnothing(get(ENV, "CI", nothing))
+
+# ## Define Parameters
+#  - Global Constants
+const FT = Float64;
+
+#  - Experiment-specific Parameters
+parameters = (
+    ## atmos parameters
+    zmin_atm = FT(0.0), # height of atm stack bottom [m] 
+    zmax_atm = FT(1.0), # height of atm stack top [m]
+    n = 15,  # number of elements in atm stack 
+    μ = FT(0.0001), # diffusion coefficient [m^2 / s]
+    T_top = FT(280.0), # fixed temperature at the top of the domain_atm [K]
+    ## slab ice parameters
+    ρc_ml = FT(4e6), # for mixed layer
+    F_basal = FT(120), # W / m2 / K
+    T_base = FT(273.16), # K 
+    L_ice = FT(3e8), # J / m3
+    h_ml = FT(1), # m 
+    T_freeze = FT(273.16), # K 
+    k_ice = FT(2), # W / m / K
+    ## coupling parameters 
+    λ = FT(1e6), # transfer coefficient 
+)
+
+# ## Define Model Functions
+
+# - Model 1 (atm) Equations
+"""
+    ∑tendencies_atm!(du, u, (parameters, T_sfc), t)
+
+Heat diffusion equation
+    dT/dt =  ∇ μ ∇ T
+    where
+        T  = 280 K              at z = zmax_atm
+        dT/dt = - ∇ F_sfc       at z = zmin_atm
+
+We also use this model to calculate and accumulate the downward surface fluxes, F_sfc:
+    F_sfc = - λ * (T_sfc - T1) 
```

Comment:
Im a little confused why F_sfc is not F_rad+SHF + LHF here.

Reply:
ah OK, I see that this is a to-do item, and that F_sfc is consistent with the ice model (but not as described in the docs for the slab ice model). Maybe you could note in the documentation that as a stand-in for F_sfc, you use this expression?